apis/swagger.yml

```diff
@@ -925,6 +925,10 @@ definitions:
         type: "array"
         items:
           type: "string"
+      EnableLxcfs:
```

Comment:
I think we need to add this parameter in API change to show the difference. Just record this and do this in the future.